File: app/src/main/java/rus/tutby/adapters/provideradapters/OnlinerPagerAdapter.java
package rus.tutby.adapters.provideradapters;

import android.support.v4.app.FragmentManager;

public class OnlinerPagerAdapter extends ProviderPagerAdapter {


    public OnlinerPagerAdapter(FragmentManager fm, String[] categories, String[] urls) {
        super(fm, categories, urls);
    }

    @Override
    public int getCount() {
        return 5;
    }
}

File: app/src/main/java/rus/tutby/ui/adapters/provideradapters/OnlinerPagerAdapter.kt
package rus.tutby.ui.adapters.provideradapters

import android.support.v4.app.FragmentManager

class OnlinerPagerAdapter(fm: FragmentManager, categories: Array<String>,
                          urls: Array<String>) : ProviderPagerAdapter(fm, categories, urls) {

    override fun getCount(): Int = 5
}
